spec/features/courses/show_spec.rb

```diff
@@ -229,6 +229,10 @@
       expect(course_page).to have_locations_map
 
       expect(course_page).to have_course_advice
+
+      expect(course_page).to have_content("Apply for this course")
```

Comment:
can we test the actual url as that will be the cta that the user will be going to

Reply:
Done 👍